### PR TITLE
Issue/1616

### PR DESCRIPTION
**Pull request by mehul0810**
## Description
This PR is for #1616.

## How Has This Been Tested?
Tested this by sorting with `Total Donated` under `Reports > Donation Methods`

## Screenshots (jpeg or gifs if applicable):
image

## Types of changes
Bug fix (non-breaking change which fixes an issue) 

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code follows has proper inline documentation.

### CODE REVIEW

File: includes/admin/reporting/class-gateways-reports-table.php [hunk 1]
```diff
@@ -153,6 +166,30 @@ protected function display_tablenav( $which ) {
...
+	 *
+	 * @param $a
+	 * @param $b
+	 *
+	 * @access public
+	 * @since  1.8.12
+	 *
+	 * @return int
+	 */
+	public function give_usort_reorder( $a, $b ) {
```
Comment (ravinderk) [inline includes/admin/reporting/class-gateways-reports-table.php]: @mehul0810 code is not format properly.

Comment (mehul0810) [inline includes/admin/reporting/class-gateways-reports-table.php]: @ravinderk Updated Code

File: includes/admin/reporting/class-gateways-reports-table.php [hunk 2]
```diff
@@ -153,6 +166,30 @@ protected function display_tablenav( $which ) {
 		<?php
 	}
 
+	/**
+	 * Reorder User Defined Array
+	 *
+	 * @param $a
```
Comment (ravinderk) [inline includes/admin/reporting/class-gateways-reports-table.php]: @mehul0810 Wrong params

File: includes/admin/reporting/class-gateways-reports-table.php [hunk 3]
```diff
@@ -153,6 +166,30 @@ protected function display_tablenav( $which ) {
...
+	 * @param $b
+	 *
+	 * @access public
+	 * @since  1.8.12
+	 *
+	 * @return int
+	 */
+	public function give_sort_total_donations( $old_value, $new_value ) {
+		// If no sort, default to label.
+		$orderby = (!empty($_REQUEST['orderby'])) ? $_REQUEST['orderby'] : 'label';
```
Comment (ravinderk) [inline includes/admin/reporting/class-gateways-reports-table.php]: @mehul0810 code formatting is not fine for this function

**Comment by DevinWalker** [comment]
Thanks @mehul0810

**Pull request merged**